examples/web-wagmi/package.json

```diff
@@ -3,8 +3,9 @@
   "private": true,
   "version": "0.1.0",
   "scripts": {
+    "build:with-packages": "turbo run build --filter=...example-wagmi",
```

Comment:
what does `--filter=...example-wagmi` do?

Reply:
Builds all dependencies of the `example-wagmi` inside monorepo, we could also build the whole monorepo but this one is better as only builds what `example-wagmi` is actually using.